amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2584,6 +2585,50 @@ VX_API_ENTRY vx_node VX_API_CALL vxExtRppSpectrogram(vx_graph graph, vx_tensor p
     return node;
 }
 
+VX_API_ENTRY vx_node VX_API_CALL vxExtRppNonSilentRegion(vx_graph graph, vx_tensor pSrc, vx_tensor pSrcLength, vx_tensor pDst1, vx_tensor pDst2, vx_scalar cutOffDB, vx_scalar referencePower, vx_scalar windowLength, vx_scalar resetInterval) {
```

Comment:
In declaration it is mentioned as pSrcRoi and here it is mentioned as pSrcLength
Please check and stick one name which is valid

Reply:
its roi - changed